src/tds/codec/token/token_col_metadata.rs

```diff
@@ -176,10 +176,6 @@ impl BaseMetaDataColumn {
 
         match ty {
             TypeInfo::VarLenSized(VarLenType::Text, _, _) => {
-                src.read_u16_le().await?;
```

Comment:
I wrote these thinking why on earth we have u16 + u16 + u8 before the table name. Well, of course I should read these in the `type_info` because that's the collation data: u32 + u8...